packages/typescript/smoke-tests/e2e/src/routes/index.tsx

```
@@ -1,12 +1,118 @@
-import { useState } from 'react'
+import { useState, useMemo } from 'react'
```

ESLint

[error] Member 'useMemo' of the import declaration should be sorted alphabetically.

(sort-imports)